packages/fxa-react/lib/AppLocalizationProvider.tsx

```diff
@@ -96,13 +95,7 @@ export default class AppLocalizationProvider extends Component<Props, State> {
   async componentDidMount() {
     const { baseDir, userLocales, bundles } = this.state;
 
-    const currentLocales = negotiateLanguages(
-      [...userLocales],
-      [...EN_GB_LOCALES, ...availableLocales],
-      {
-        defaultLocale: 'en-US',
-      }
-    );
+    const currentLocales = parseAcceptLanguage(userLocales.join(', '));
```

Comment:
👌 

Will be nice when we can go back and clean this up more in that modernize localization epic but I'm glad just pulling in this fixes it.

Reply:
Agreed, super simple fix for now though. Thanks for talking through this on slack with me.